### PR TITLE
In a directory listing, adds the media type class when we know it 

### PR DESCRIPTION
**Pull request by timbl**
In a directory listing, adds the media type class when we know it  for a directory member.   Add sdct:modified with an RDF date, which IMHO should deprocate the mtime returned as a unixinteger. (The RDF way is when know the data type and RDF has it, you use it. Note the code is actually shorter, internally node uses s Date() object)

Addresses Issue https://github.com/solid/node-solid-server/issues/446

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
@timbl I fixed the jsStandard formatting errors that were failing the test and pushed to this PR. Now there's some tests failing about container serialization; investigating.

**Comment by timbl** [comment]
@dmitrizagidulin it is exactly th container list test  which I would expect to fail, as that is what has changed.  I need to understand the proper way of running the tests ....

**Comment by dmitrizagidulin** [comment]
@timbl the easiest way to run the tests is to just invoke `npm test` on the command line. (That also runs the Standard linter).

File: test/ldp.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ describe('LDP', function () {
         '   dcterms:title "This is a magic type" ;' +
         '   o:limit 500000.00 .', 'sampleContainer/magicType.ttl')
 
-      ldp.listContainer(path.join(__dirname, '/resources/sampleContainer/'), 'https://server.tld', '', 'text/turtle', function (err, data) {
+      ldp.listContainer(path.join(__dirname, '/resources/sampleContainer/'), 'https://server.tld/resources/sampleContainer/', 'https://server.tld', '', 'text/turtle', function (err, data) {
```
Comment (nicola) [inline test/ldp.js]: Isn't this the equivalent of just passing "/resources/sampleContainer/" ?

**Comment by bshambaugh** [comment]
I just figured out you're using Jasmine in ldp.js :) https://jasmine.github.io/2.0/introduction.html

**Pull request merged**